Directory.Build.props

```diff
@@ -31,6 +31,7 @@
     <LangVersion>latest</LangVersion>
     <IsShipping>false</IsShipping>
     <WarnOnPackingNonPackableProject>false</WarnOnPackingNonPackableProject>
+    <DebugType>Full</DebugType>
```

Comment:
The weirdest thing is that I don't understand how before this prop would break debugging of BarViz. 

Its a compiler option for generating debugging information - https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation#debugtype

It makes sense to have it here and it makes sense that we would have issues debugging certain projects like BarViz without it. But why did it break debugging during Hackathon I don't really know.

Maybe Visual Studio performs some extra setup that was conflicting with this prop and then once we moved the project to the main solution the setup changed?

Reply:
Can you put this information in a comment next to the property?